docs/contributing-to-analyses/analysis-modules/modifying-a-module.md

```diff
@@ -0,0 +1,19 @@
+# Modifying an existing module
+
+When modifying or adding to an existing module, it's important to document all changes you make.
+
+!!! note
+    When you file an issue to track your work modifying an analysis module, you will use the [`Update an analysis` issue template](../../communications-tools/github-issues/issue-templates.md#update-an-analysis).
+
+Please make sure to make the following updates when you work with an existing module:
```

Comment:
```suggestion

If you are modifying a module that already exists, then you will first need to file an issue using the [`Update an analysis` issue template](../../communications-tools/github-issues/issue-templates.md#update-an-analysis). 
Then you can create a new branch and commit any code changes. 

Be sure that any new code changes are accompanied by updates to documentation. 
You will also want to capture any new software dependencies. 

The following changes should be included in any PR's that modify existing modules: 
```
I felt like the first sentence was repetitive and only focused on documentation when that's not the only thing captured here. Also not sure this is the right place for the note? I think that's a first step before doing any of this so I moved it around.

Reply:
> Also not sure this is the right place for the note? 

I too was not sure fwiw!